app/views/vacancies/index.html.haml

```diff
@@ -1,4 +1,10 @@
 - content_for :page_title_prefix, t('jobs.heading')
+- unless (path_to_next_page @vacancies).nil?
+  = content_for :for_head_link do
```

Comment:
I find that `content_for :for_head_link` reads a bit odd, the double for especially.  `content_for :head_links` maybe?

Reply:
Thank you, I really like the suggested name so I've changed it in https://github.com/DFE-Digital/teacher-vacancy-service/pull/983/commits/6ce0868d306fb803f739a17b146644b12c153e2b